server/src/main/scala/ch.epfl.scala.index.server/DataRepository.scala

```diff
@@ -89,8 +90,13 @@ class DataRepository(
               current = clamp(page),
               totalPages = Math.ceil(r.totalHits / params.total.toDouble).toInt,
               total = r.totalHits
-            ),
-            r.to[Project].toList.map(hideId)
+            ), {
+              // println(r.hits.toList.take(100) map { h =>
+              //   val source = h.sourceAsMap
+              //   s"""${h.score}: ${source("organization")}/${source("repository")}"""
+              // })
```

Comment:
These debug statements probably shouldn't be there.

Reply:
indeed